tests/rich-text.spec.ts

```diff
@@ -0,0 +1,34 @@
+import { press } from '@blocksuite/virgo/tests/utils/misc';
+import { expect, test } from '@playwright/test';
+
+import {
+  enterPlaygroundRoom,
+  focusRichText,
+  initEmptyParagraphState,
+  type,
+} from './utils/actions/index.js';
+
+test('ZERO_WIDTH_SPACE should be counted by one cursor position', async ({
+  page,
+}) => {
+  await enterPlaygroundRoom(page);
+  await initEmptyParagraphState(page);
+  await focusRichText(page);
+  await page.waitForTimeout(100);
+  await page.keyboard.down('Shift');
+  await press(page, 'Enter');
+  await page.keyboard.up('Shift');
+  await page.waitForTimeout(100);
+  await type(page, 'asdfg');
+  await page.waitForTimeout(100);
+  await press(page, 'Enter');
+  await page.waitForTimeout(100);
+  await page.keyboard.down('Meta');
```

Comment:
The testing platform is Linux, so there is no Meta key.

You can use `SHORT_KEY` instead, Or you can use the following pattern

`await page.keyboard.press(`${SHORT_KEY}+Enter`, { delay: 50 });`